### PR TITLE
Feature/change settings

### PR DESCRIPTION
**Pull request by spyralspy01**
- Created change_username function to update the database.
- Created settings page in layout which redirects to username change.

### CODE REVIEW

**Comment by spyralspy01** [comment]
-- Already layout.html is in develop branch 
-- Uploaded in this feature branch for testing of its functionality.
will resolve conflicts later.

File: flask-discord/flaskd/auth.py [hunk 1]
```diff
@@ -9,6 +9,17 @@
 from flaskd.db import get_db
 
 bp = Blueprint('auth',__name__,url_prefix='/auth')
+#
+@bp.route("/settings",methods=("GET","POST"))
+def change_username():
+    if request.method == "POST":
+        new_username=request.form["uname"]
+        email=request.form['email']
+        # print(new_username)
```
Comment (Nithish4603) [inline flask-discord/flaskd/auth.py:18]: Good code simple and works as intended.

**Review by Nithish4603 — COMMENTED** [review]
Good work finished the tasks on time before schedule.

File: flask-discord/flaskd/templates/settings.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+    <body>
+        <form action="/auth/settings" method="post">
+            <div class="mb-3">
+              <label for="Change Username" class="form-label"> Chnage Username</label>
```
Comment (Sathvik-Chowdary-Veerapaneni) [inline flask-discord/flaskd/templates/settings.html:6]: - **Chnage Username** need to correct into **Change Username**

**Review by Sathvik-Chowdary-Veerapaneni — APPROVED** [review]
Everything is working well.
Note for next release
- required bit more changes to UI design

**Review by Nithish4603 — APPROVED** [review]

**Pull request merged**